services/order.py

```diff
@@ -0,0 +1,41 @@
+from datetime import datetime
+from django.db import transaction
+from django.db.models import QuerySet
+from django.contrib.auth import get_user_model
+from db.models import Order, Ticket, MovieSession
+
+User = get_user_model()
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> Order:
+    user = User.objects.get(username=username)
+
+    if date:
+        created_at = datetime.strptime(date, "%Y-%m-%d %H:%M")
+    else:
+        created_at = datetime.now()
+    order = Order.objects.create(user=user, created_at=created_at)
```

Comment:
Here you should not pass created_at - it should be populated automatically if you add auto_now_add.
Then, if date is being passed you can change it on order object